tiflash/tiflash-configuration.md

```diff
@@ -30,25 +30,67 @@ aliases: ['/docs-cn/dev/tiflash/tiflash-configuration/','/docs-cn/dev/reference/
 ### 配置文件 tiflash.toml
 
 ```toml
-tmp_path = tiflash 临时文件存放路径
-path = tiflash 数据存储路径     # 如果有多个目录，以英文逗号分隔
-path_realtime_mode = false # 默认为 false。如果设为 true，且 path 配置了多个目录，表示在第一个目录存放最新数据，较旧的数据存放于其他目录。
-listen_host = tiflash # TCP/HTTP 等辅助服务的监听 host，建议配置成 0.0.0.0
-tcp_port = tiflash tcp 服务端口
-http_port = tiflash http 服务端口
-mark_cache_size = 5368709120 # 数据块元信息的内存 cache 大小限制，通常不需要修改
-minmax_index_cache_size = 5368709120 # 数据块 min-max 索引的内存 cache 大小限制，通常不需要修改
-```
+# tiflash TCP/HTTP 等辅助服务的监听 host，建议配置成 0.0.0.0
+listen_host = "0.0.0.0"
+## tiflash tcp 服务端口
+tcp_port = 9000
+## tiflash http 服务端口
+http_port = 8123
+## 数据块元信息的内存 cache 大小限制，通常不需要修改
+mark_cache_size = 5368709120
+## 数据块 min-max 索引的内存 cache 大小限制，通常不需要修改
+minmax_index_cache_size = 5368709120
+
+## tiflash 数据存储路径，如果有多个目录，以英文逗号分隔。
+## 从 v4.0.9 版本开始 path 及 path_realtime_mode 参数不再推荐使用，推荐使用 [storage] 下的配置项代替，在多盘部署的情况下能更好地利用节点性能。
+# path = "/tidb-data/tiflash-9000"
+## 或
+# path = "/ssd0/tidb-data/tiflash,/ssd1/tidb-data/tiflash,/ssd2/tidb-data/tiflash"
+## 默认为 false。如果设为 true，且 path 配置了多个目录，表示在第一个目录存放最新数据，较旧的数据存放于其他目录。
+# path_realtime_mode = false
+
+## tiflash 临时文件存放路径
+# tmp_path = "/tidb-data/tiflash-9000/tmp"
+
+## 存储路径相关配置， 从 v4.0.9 开始生效
+[storage]
+    ## 如果节点上有多块 SSD 硬盘，推荐一个数据存储目录对应一块硬盘，并把数据目录填到列表 `storage.main.dir` 中，
+    ## 以更好地利用节点性能。
+
+    ## 如果节点上有多块 IO 性能相差较大的硬盘（如一块 SSD 硬盘加上多块 HDD 硬盘），如下配置可以更好地利用节点性能。
+    ## * 把 SSD 硬盘的数据存储目录配置在 `storage.latest.dir` 中
+    ## * 把 HDD 硬盘的数据存储目录配置在 `storage.main.dir` 中
+
+    [storage.main]
+    ## 用于存储主要数据的目录列表
+    dir = [ "/tidb-data/tiflash-9000" ] 
+    ## 或
+    # dir = [ "/ssd0/tidb-data/tiflash", "/ssd1/tidb-data/tiflash" ]
+
+    ## storage.main.dir 存储目录列表中，每个目录的最大可用容量。
+    ## * 在未定义配置项，或者列表中全填 0时，会使用目录所在的硬盘容量
+    ## * 以 byte 为单位，目前不支持如 "10GB" 类似的设置
+    ## * capacity 列表的长度应当与 dir 长度保持一致
+    ## 例如：
+    # capacity = [ 10737418240, 10737418240 ]
+
+    [storage.latest]
+    ## 用于存储新数据的目录列表。
+    ## 在未配置或者为空列表的情况下，会使用 `storage.main.dir` 的值。
+    # dir = [ ]
+    ## storage.latest.dir 存储目录列表中，每个目录的最大可用容量。
+    # capacity = [ 10737418240, 10737418240 ]
+
+    [storage.raft]
+    ## 用于存储 Raft 数据的目录列表。
+    ## 在未定义配置项或者为空列表的情况下，会以 `storage.latest.dir` 的列表目录中每一项添加 "/kvstore" 作为此项的值。
+    # dir = [ ]
```

Comment:
```suggestion
```

I don't think we should expose this configuration to users. They are 99.9% use the same path as `storage.latest.dir`.